pkg/phases/photon.go

```diff
@@ -0,0 +1,28 @@
+package phases
+
+import (
+	"github.com/flanksource/konfigadm/pkg/types"
+	"github.com/flanksource/konfigadm/pkg/utils"
+)
+
+var Photon = photon{}
+
+type photon struct {
+}
+
+func (p photon) GetPackageManager() types.PackageManager {
+	return YumPackageManager{}
+}
+
+func (p photon) GetTags() []types.Flag {
+	return []types.Flag{types.PHOTON}
```

Comment:
I think photon is also redhat-like

Reply:
Yeah, I first also did that but some packages are not present in photon which are present in redhat-like for ex: `device-mapper-persistent-data`, after that I changed removed the flag and kept it separate.